### PR TITLE
Slightly faster deployment

### PR DESCRIPTION
**Pull request by celskeggs**
This should hopefully be easy to review. Not yet tested, but will be shortly.

### CODE REVIEW

File: building/build-debs/homeworld-admin-tools/src/seq.py [hunk 1]
```diff
@@ -62,45 +64,84 @@ def ver():
...
+    print("    $ spire deploy dns-addon")
     ops.add_operation("deploy or update dns-addon", lambda: deploy.launch_spec("dns-addon.yaml"))
+    print("  to verify dns-addon:")
+    print("    $ spire verify dns-addon-run")
     ops.add_operation("verify that dns-addon is online", iterative_verifier(verify.check_dns_kubeinfo, 60.0))
+    print("    $ spire verify dns-addon-query")
+    ops.add_operation("verify that dns-addon is functioning", verify.check_dns_function)
+
+
+def sequence_addons(ops: setup.Operations) -> None:
```
Comment (krawthekrow) [inline building/build-debs/homeworld-admin-tools/src/seq.py]: Should reuse sequence_flannel and sequence_dns_addon as in sequence_supervisor.

Comment (celskeggs) [reply building/build-debs/homeworld-admin-tools/src/seq.py]: No -- that's purposeful. They should be interleaved for the sake of spending less time waiting for deployment.

Comment (krawthekrow) [reply building/build-debs/homeworld-admin-tools/src/seq.py]: Alright, if that's intentional. It's minor anyway if the verification steps will ultimately be isolated as per #172.

**Comment by celskeggs** [comment]
I've provided a proposed solution. It's not quite right, given that it involves doing some hacky things with storing attributes on functions, but it seems to work, and I think the model is otherwise good. Thoughts?

**Comment by celskeggs** [comment]
Ah, woops, I accidentally merged the first half of this PR. Hopefully that works out, since that part was approved and seemed uncontentious. The contentious part is still here in this PR.

(I would roll it back, but I figure it's a lesser evil to leave it as-is than to rewrite the history of the master branch.)

**Comment by celskeggs** [comment]
Because we don't currently have infrastructure for handling flags in spire (#181), I'll put those off until later. Other changes put in #182.

I'm going to test this, and then merge it if it passes.

**Pull request merged**